src/components/file-upload/index.md.njk

```diff
@@ -23,6 +23,54 @@ There are 2 ways to use the file upload component. You can use HTML or, if you
 
 {{ example({group: "components", item: "file-upload", example: "default", html: true, nunjucks: true, open: false, id: "default-2"}) }}
 
+### Error messages
+
+Error messages should be styled like this:
+
+{{ example({group: "components", item: "file-upload", example: "error", html: true, nunjucks: true, open: false, size: "m"}) }}
+
+Make sure errors follow the guidance in [error message](/components/error-message/) and have specific error messages for specific error states.
+
+#### If no file has been selected
+Say ‘Select a [whatever they need to select]’.<br>
+For example, ‘Select a report’.
+
+#### If the file is the wrong file type
+Say‘The [whatever] must be a [list of file types]’.<br>
+For example, ‘The report must be a CSV or ODS’ or ‘The selected file must be a JPG, BMP, PNG, TIF or PDF’.
+
+#### If the file is too big
+Say ‘The [whatever] must be smaller than [largest file size]’.<br>
+For example, ‘The CSV must be smaller than 2MB’ or ‘The selected file must be smaller than 10MB’.
+
+#### If the file is empty
+Say ‘The [whatever] is empty’.<br>
+For example, ‘The CSV is empty’ or ‘The selected file is empty’.
+
+#### If the file contains a virus
+Say ‘The [whatever] contains a virus’.<br>
+For example, ‘The document contains a virus’ or ‘The selected file contains a virus’.
+
+#### If the file is password protected
+Say ‘The [whatever] is password protected’.<br>
+For example, ‘The report is password protected’ or ‘The selected file is password protected’.
+
+#### If there was a problem and the file was not uploaded
+Say ‘The [whatever] could not be uploaded – try again’.<br>
+For example, ‘The CSV could not be uploaded – try again’ or ‘The selected file could not be uploaded – try again’.
+
+#### If the user can select more than one file but there is a limit on how many they can select
```

Comment:
Do we need to say 'If the user can select more than one file'? Couldn't we simply say 'If there is a limit on how many files a user can select' to make it more concise?

Reply:
Agreed.